src/main/java/seedu/duke/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.duke;
+
+/**
+ * Represents an <code>Exception</code> that occurs when Duke is running.
+ */
+public class DukeException extends Exception {
+    public DukeException(String message) {
+        super(message);
+    }
+}
```

Comment:
Not sure I can agree on this abstraction. Wouldn't this make it more difficult to locate the cause of exceptions?